src/lib.rs

```diff
@@ -96,8 +96,8 @@ impl Aviffy {
         let ispe_prop = ipco.push(IpcoProp::Ispe(IspeBox { width, height }));
         // This is redundant, but Chrome wants it, and checks that it matches :(
         let av1c_prop = ipco.push(IpcoProp::Av1C(Av1CBox {
-            seq_profile: false,
-            seq_level_idx_0: 0,
+            seq_profile: if twelve_bit { 2 } else { 1 },
+            seq_level_idx_0: 31,
```

Comment:
I inspected the rav1e source code and found that it currently uses a hardcoded value of 31 for the `seq_level_idx[ 0 ]`  syntax element. In rav1e/src/encoder.rs, we have:

```
    if fi.sequence.reduced_still_picture_hdr {
      assert!(!fi.sequence.timing_info_present);
      assert!(!fi.sequence.decoder_model_info_present_flag);
      assert_eq!(fi.sequence.operating_points_cnt_minus_1, 0);
      assert_eq!(fi.sequence.operating_point_idc[0], 0);
      self.write(5, 31)?; // level
      assert_eq!(fi.sequence.tier[0], 0);
    } else {
      self.write_bit(fi.sequence.timing_info_present)?; // timing info present

      if fi.sequence.timing_info_present {
        self.write(32, fi.sequence.time_base.num)?;
        self.write(32, fi.sequence.time_base.den)?;

        self.write_bit(true)?; // equal picture interval
        self.write_bit(true)?; // zero interval
        self.write_bit(false)?; // decoder model info present flag
      }

      self.write_bit(false)?; // initial display delay present flag
      self.write(5, 0)?; // one operating point
      self.write(12, 0)?; // idc
      self.write(5, 31)?; // level
      self.write(1, 0)?; // tier
    }
```

Note the two `self.write(5, 31)?; // level` lines